src/DynamoCoreWpf/ViewModels/Preview/ConnectorContextMenuViewModel.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System.Windows;
+using System.Windows.Threading;
+using Dynamo.Core;
+using Dynamo.UI.Commands;
+using System;
+
+namespace Dynamo.ViewModels
+{
+    public class ConnectorContextMenuViewModel : NotificationObject
+    {
+        #region Properties
+        private Point currentPosition;
+        private bool isCollapsed;
+        private ConnectorViewModel ViewModel { get; set; }
+        private Dispatcher Dispatcher { get; set; }
+
+        /// <summary>
+        /// Controls connector visibility: on/off. When wire is off, additional styling xaml turns off tooltips.
+        /// </summary>
+        public bool IsCollapsed
```

Comment:
It's worth to create some unit tests to guard the initialization and assert public properties given this is a new class.

Reply:
Done (y)